Example/chatHeads/index.js

```diff
@@ -0,0 +1,250 @@
+import React, { Component } from 'react';
+import { Dimensions, Image, View, StyleSheet } from 'react-native';
+import Animated from 'react-native-reanimated';
+
+import { PanGestureHandler, State } from 'react-native-gesture-handler';
+
+const { width } = Dimensions.get('window');
+
+const {
+  set,
+  neq,
+  cond,
+  eq,
+  add,
+  multiply,
+  lessThan,
+  spring,
+  block,
+  startClock,
+  stopClock,
+  clockRunning,
+  sub,
+  defined,
+  Value,
+  Clock,
+  event,
+} = Animated;
+
+function follow(clock, value) {
+  const config = {
+    damping: 28,
+    mass: 0.3,
+    stiffness: 188.296,
+    overshootClamping: false,
+    toValue: value,
+  };
+
+  const state = {
+    finished: new Value(0),
+    velocity: new Value(0),
+    position: new Value(0),
+    time: new Value(0),
+  };
+  return block([
+    cond(clockRunning(clock), 0, startClock(clock)),
```

Comment:
we never stop this clock. I think it would be easier to just use a single clock instead of clock and clock2. Then we should test if all spring animations are complete before we stop it